[Frontend] The docs for Encore setup simply do not work and error due to package.json not being populated

Hello,

The documentation says to install Encore then run `npm run watch`, however this does not work because no scripts get added to package.json.

Steps to reproduce.
1) https://symfony.com/doc/current/frontend/encore/installation.html
**Installing Encore in non Symfony Applications**
`npm install @symfony/webpack-encore --save-dev`

2) https://symfony.com/doc/current/frontend/encore/simple-example.html
`npm run watch`

Error:
```
λ npm run watch
npm ERR! Missing script: "watch"
npm ERR!
npm ERR! To see a list of scripts, run:
npm ERR!   npm run
```

Package.json:
```
{
  "dependencies": {
    "@symfony/webpack-encore": "^4.2.0"
  }
}
```

Doesn't matter if you use NPM or Yarn, or whatever command you use. 

Unless I am missing something this is supposed to work as it's step 1 to 2 of the documentation.

> Do I understand correctly that instructions work when using it on a standard Symfony application?

If you run it as if it's a symfony application, eg:
- `composer require symfony/webpack-encore-bundle`
- `npm install`

This also does not work, unless installing a symfony project is required, those commands on their own do not work because a package.json is not created.

```
λ npm install
npm ERR! code ENOENT
npm ERR! syscall open
npm ERR! path xxx/package.json
npm ERR! errno -4058
npm ERR! enoent ENOENT: no such file or directory, open 'xxx\package.json'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent
```

So we continue the struggle...

Clicking that blog announcement shows a github link: https://github.com/symfony-cli/symfony-cli
And on here it says: Read the installation instructions on [symfony.com](https://symfony.com/download). okay so I go there..

Not one of the "copy" buttons work by the way. Another bug to add to the list of bugs or mistakes I've found so far in this thread.. Tested Chrome and Firefox, they don't work.

Anyway, ran install again and then the `mv` command. Cool I have `Symfony CLI version 5.4.19`

**How does any of this help the original issue?**

> [Installing Encore in non Symfony Applications](https://symfony.com/doc/current/frontend/encore/installation.html#installing-encore-in-non-symfony-applications)
```
➜  test npm install @symfony/webpack-encore --save-dev
npm WARN deprecated stable@0.1.8: Modern JS already guarantees Array#sort() is a stable sort, so this library is deprecated. See the compatibility table on MDN: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#browser_compatibility

added 623 packages, and audited 624 packages in 50s

62 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
➜  test npm run watch
npm ERR! Missing script: "watch"
npm ERR!
npm ERR! To see a list of scripts, run:
npm ERR!   npm run

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/xxxx/.npm/_logs/2022-12-18T21_53_46_503Z-debug.log
➜  test cat package.json
{
  "devDependencies": {
    "@symfony/webpack-encore": "^4.2.0"
  }
}
```

I am sorry if I come across rude but I've hit 3 separate issues with the docs/symfony tools and none of this leads to **fixing the issue with package.json not updating for _NON_ symfony projects**

I shouldn't need to bother with the symfony CLI for a **NON** symfony application you never used to and the documentation doesn't say it's a mandatory requirement. Even with the updated Symfony CLI it doesn't fix the issue for **NON** symfony applications.

If i manually add `encore` commands to my package.json, it works, but the documentation doesn't say to do that and it's expected they get added automatically.

https://symfony.com/doc/current/frontend/encore/installation.html#installing-encore-in-non-symfony-applications

It says "This command creates (or modifies) a package.json file" < **This is bugged and it does not do that.**

--- a/frontend/encore/installation.rst
+++ b/frontend/encore/installation.rst
@@ -139,6 +139,9 @@ is the main config file for both Webpack and Webpack Encore:
 
     module.exports = Encore.getWebpackConfig();
 
+Creating Other Supporting File
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
 Next, open the new ``assets/app.js`` file which contains some JavaScript code
 *and* imports some CSS:
 
@@ -185,7 +188,7 @@ a system that you'll learn about soon:
     // register any custom, 3rd party controllers here
     // app.register('some_controller_name', SomeImportedController);
 
-And finally, create an ``assets/controllers.json`` file, which also fits into
+Then create an ``assets/controllers.json`` file, which also fits into
 the Stimulus system:
 
 .. code-block:: json
@@ -193,6 +196,18 @@ the Stimulus system:
     {
         "controllers": [],
         "entrypoints": []
+    }
+
+Finally, though it's optional, add the following ``scripts`` to your ``package.json``
+file so you can run the same commands in the rest of the documentation:
+
+.. code-block:: json
+
+    "scripts": {
+        "dev-server": "encore dev-server",
+        "dev": "encore dev",
+        "watch": "encore dev --watch",
+        "build": "encore production --progress"
     }
 
 You'll customize and learn more about these files in :doc:`/frontend/encore/simple-example`.

--- a/frontend/encore/simple-example.rst
+++ b/frontend/encore/simple-example.rst
@@ -154,9 +154,10 @@ template: the paths in ``entrypoints.json`` will always be the final, correct pa
 And if you use :doc:`splitEntryChunks() </frontend/encore/split-chunks>` (where Webpack splits the output into even
 more files), all the necessary ``script`` and ``link`` tags will render automatically.
 
-If you're *not* using Symfony, you can ignore the ``entrypoints.json`` file and
-point to the final, built file directly. ``entrypoints.json`` is only required for
-some optional features.
+If you are not using Symfony you won't have the ``encore_entry_*`` functions available.
+Instead, you can point directly to the final built files or write code to parse
+``entrypoints.json`` manually. The entrypoints file is needed only if you're using
+certain optional features, like ``splitEntryChunks()``.
 
 .. versionadded:: 1.9.0
 

